lib/internal/fs/utils.js

```diff
@@ -749,6 +750,7 @@ const validateCpOptions = hideStackFrames((options) => {
   validateBoolean(options.force, 'options.force');
   validateBoolean(options.preserveTimestamps, 'options.preserveTimestamps');
   validateBoolean(options.recursive, 'options.recursive');
+  validateBoolean(options.absolute, 'options.absolute');
```

Comment:
How about `options.verbatimSymlinks` instead? The name `absolute` does not say much about what the option does. I'd imagine that `verbatimSymlinks` defaults to `false` and can be set to `true` to always copy symlinks without any kind of path resolution (similar to `cp -RP`). This appears to be the behavior you are looking for.

Reply:
Makes sense! I was not very convinced by the `absolute` name, so I'm going to change it.

Reply:
Changed!